src/components/sidemeny/sidemeny.tsx

```diff
@@ -22,8 +22,10 @@ export const Sidemeny: React.FC = () => {
 
 	return (
 		<section className="sidemeny">
-			<h2 onClick={() => scrollTilElement('#veilarbpersonflatefs-root')}>Detaljer om bruker</h2>
-			<nav>
+			<nav aria-labelledby="detaljer-om-bruker">
+				<h2 onClick={() => scrollTilElement('#veilarbpersonflatefs-root')} id="detaljer-om-bruker">
```

Comment:
Kan du bruke `<Heading>` her istedenfor `h2`? Med riktig level for uu

Reply:
jeg veeet ![bitmoji](https://sdk.bitmoji.com/render/panel/82a33c72-3399-441f-8a2e-4132b1f8ce51-f0600307-636d-42ff-bf1c-39ed7447b108-v1.png?transparent=1&palette=1&width=246)